scrapinghub/client/__init__.py

```diff
@@ -43,13 +45,17 @@ class ScrapinghubClient(object):
         <scrapinghub.client.ScrapinghubClient at 0x1047af2e8>
     """
 
-    def __init__(self, auth=None, dash_endpoint=None, **kwargs):
+    def __init__(self, auth=None, dash_endpoint=None,
+                 connection_timeout=DEFAULT_CONNECTION_TIMEOUT, **kwargs):
```

Comment:
`HubstorageClient` always sets a connection timeout even if you pass `None` or `0`, I think we should do the same as https://github.com/scrapinghub/python-scrapinghub/blob/master/scrapinghub/hubstorage/client.py#L82

Reply:
Good point, I did it in this way initially but then decided to "optimize" the changes, thanks 🙂